agenta-cli/agenta/client/client.py

```diff
@@ -1,112 +1,47 @@
-from typing import Dict, Any, Optional
 import os
+import toml
 import time
 import click
+from typing import Dict
 from pathlib import Path
-from typing import List, Optional, Dict, Any
-
-import requests
-from agenta.client.api_models import AppVariant, Image, VariantConfigPayload
-from docker.models.images import Image as DockerImage
+from agenta.client.backend import client
+from agenta.client.api_models import Image
 from requests.exceptions import RequestException
 
-BACKEND_URL_SUFFIX = os.environ.get("BACKEND_URL_SUFFIX", "api")
-
-
-class APIRequestError(Exception):
-    """Exception to be raised when an API request fails."""
-
 
-def get_base_by_app_id_and_name(
-    app_id: str, base_name: str, host: str, api_key: str = None
-) -> str:
-    """
-    Get the base ID for a given app ID and base name.
-
-    Args:
-        app_id (str): The ID of the app.
-        base_name (str): The name of the base.
-        host (str): The URL of the server.
-        api_key (str, optional): The API key to use for authentication. Defaults to None.
-
-    Returns:
-        str: The ID of the base.
+class ClientWrapper:
+    def __init__(self):
```

Comment:
instead of call the init with the param you need. don't use the env or config.toml here

Reply:
after removing the logic for config stuff, there is no need for this class at all.